src/field/username.js

```diff
@@ -55,5 +55,5 @@ export function setUsername(m, str, usernameStyle = 'username', validateUsername
 }
 
 export function usernameLooksLikeEmail(str) {
-  return str.indexOf('@') > -1;
+  return str.indexOf('@') > -1 && str.indexOf('.') > -1;
```

Comment:
Can you please use a more robust email validation library?
Ideally, the same library used in the backend should be used in Lock: 
https://github.com/validatorjs/validator.js#client-side-usage

Reply:
this validation is only for cases that the browser doesn't support html5 validation, since we use that by default. we're not adding +20kb just to do email validation 😬